src/main/java/step4/model/CarFactory.java

```diff
@@ -0,0 +1,28 @@
+package step4.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarFactory {
+
+  private static final String VARIABLE_NAME_SPLIT = ",";
+
+  private CarFactory(){}
+
+  public static Cars makeCars(String carNames) {
+    return makeCarsWithMinimumLocs(carNames, 0);
+  }
+
+  public static Cars makeCarsWithMinimumLocs(String carNames, int minimumLoc){
+    List<Car> carList = new ArrayList<>();
+    String[] carNameArr = splitCars(carNames);
+    for(String name: carNameArr) carList.add(new Car(name, minimumLoc));
```

Comment:
```suggestion
    for(String name: carNameArr) {
      carList.add(new Car(name, minimumLoc));
    }
```
한 줄 밖에 없더라도 블록으로 감싸주세요!